bin/bbrelease

```diff
@@ -239,7 +236,9 @@ if [ X$revmode = Xyes ]; then
     # Copy old version of config/lt_vers.am, since it's hard to
     # "undo" changes to it.
     cp config/lt_vers.am $tmpdir
-    branch=`git symbolic-ref -q --short HEAD`
+    if [ "${BRANCHNAME}" = "" ]; then
+        branch=`git symbolic-ref -q --short HEAD`
```

Comment:
```suggestion
        branch=`git symbolic-ref -q --short HEAD`
    else
        branch=${BRANCHNAME}
```
if BRANCHNAME is passed in with --branch, the value will be needed in $branch in line 247.